### PR TITLE
Moves a bunch of mining items from Science research to the Mining points vendor.

### PR DESCRIPTION
**Pull request by athena413**
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

also it makes the "Advanced Sanitation Technology" tree's description actually be a proper daft punk reference

EVERY CHANGE I MADE

KA Range Increase, Damage Increase, Cooldown Decrease: 1000 -> 300

KA AOE Damage: 2000 -> 750

JUSTIFICATION: If miners are going to be mining for more than like 10 minutes with a kinetic accelerator, let's at least make it so that it doesnt cost 3000 fucking points to get the upgrades.

ADDED Basic Plasma Cutter: 3500 points

JUSTIFICATION: While it's "basic", it's still pretty damn good. I don't see the point in making it an uber-cheapo option, give it some progression at least.

ADDED Bluespace Ore Satchel of Holding: 4000 points

JUSTIFICATION: It's quite a huge convenience for miners, and this feels like a decent enough price for it. I don't see the point in making it cost like 10000 because this _is_ a really nice convenience and it would just not be good to have to get that many points for a good bag

ADDED Advanced Plasma Cutter: 5000 points

JUSTIFICATION: For what it is, I think this is a reasonable price for it. It's 1500 more points than the basic one.

TECH TREES:

BASIC MINING: REMOVED KINETIC ACCELERATOR DAMAGE, COOLDOWN, AND RANGE MOD, + PLASMA CUTTER

ADVANCED MINING: REMOVED ADVANCED PLASMA CUTTER

APPLIED BLUESPACE: REMOVED BLUESPACE SATCHEL OF HOLDING

Mining AoE mod remains researchable and printable.

### Why is this change good for the game?

Almost fully removes the Science and Mining interaction, since it's toxic as hell and all attempts to "fix" the interaction have failed massively,

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms.
See below

### What should players be aware of when it comes to the changes your PR is implementing?
YOU CAN NO LONGER PRINT A GOOD AMOUNT OF MINING EQUIPMENT
ITS MOSTLY ALL IN THE POINTS VENDOR NOW

### What general grouping does this PR fall under? 
Research tech update, Mining vendor updates

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
N/A

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
See "Intent of your Pull Request"

# Changelog

:cl:  
rscdel: Removed multiple items from Advanced Mining, Mining Tech, and Applied Bluespace
rscadd: Anything removed from the above has been added to the Mining points vendor.
/:cl:

### CODE REVIEW

**Review by redmoogle — CHANGES_REQUESTED** [review]
Leave a way for golems and freeminers to get this as well since they both can make them (they have special vendor types because ofc)

**Comment by athena413** [comment]
> 
> 
> Leave a way for golems and freeminers to get this as well since they both can make them (they have special vendor types because ofc)

freeminers have it now

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -276,6 +279,7 @@
 		new /datum/data/mining_equipment("Silver Pickaxe",				/obj/item/pickaxe/silver,										750, VENDING_WEAPON),
 		new /datum/data/mining_equipment("Diamond Pickaxe",				/obj/item/pickaxe/diamond,										1500, VENDING_WEAPON),
 		new /datum/data/mining_equipment("Plasma Cutter" ,				/obj/item/gun/energy/plasmacutter,								2500, VENDING_WEAPON),
+		new /datum/data/mining_equipment("Advanced Plasma Cutter",		/obj/item/gun/energy/plasmacutter/adv,							5000, VENDING_WEAPON),
 		new /datum/data/mining_equipment("Plasma Cutter Shotgun",		/obj/item/gun/energy/plasmacutter/scatter,						6000, VENDING_WEAPON),
 		new /datum/data/mining_equipment("Plasma Shotgun Upgrade",		/obj/item/upgrade/plasmacutter/defuser,							1000, VENDING_WEAPON),
 		new /datum/data/mining_equipment("KA Minebot Passthrough",		/obj/item/borg/upgrade/modkit/minebot_passthrough,				100, VENDING_UPGRADE),
```
Comment (redmoogle) [inline code/modules/mining/machine_vending.dm:285]: Dont forget to apply the lowering cost if you want to

**Review by redmoogle — APPROVED** [review]
Code good

**Comment by alexkar598** [comment]
Giving miners the advanced plasmacutter without science's help is simply powercreep. The basic plasmacutter is more than sufficient at making mining's job not suck ass. The advanced one is simply overkill

**Comment by alexkar598** [comment]
Same thing for the bluespace mining satchel. You don't need it to have a somewhat efficient time as miner. Its the kinda of QOL shit that's better left to upgrades like how the AEG is an upgrade that science can research for security. Your job is somewhat harder without but its still fun.

**Comment by athena413** [comment]
> 
> 
> Giving miners the advanced plasmacutter without science's help is simply powercreep. The basic plasmacutter is more than sufficient at making mining's job not suck ass. The advanced one is simply overkill

I completely agree, but I think that at this point it's better to do this than to simply let the entire mining-science toxicity fester and explode like it has been for at least a year now. If I keep the advanced cutter or the satchel of holding as a research item the interaction will remain and stay as toxic as it is currently.

Theos has something in the works as at least an idea to move the job of getting the cutters and satchel to Cargo but until then I'm tired of hearing shit about miners dumping the ORM because Science wanted to do something else that round.

**Comment by Marmio64** [comment]
maybe lower the basic plasmacutter's price a bit, 3.5k mining points takes quite a while to get with just a PKA.

**Comment by Mesalikepie** [comment]
This is actually dumb. This isn't a problem that needs to be solved. The relationship between science and mining on 90% of rounds is not toxic, with mining getting ores, science getting tech. It works because it's a game and the two departments have to work together to prosper. Cause you know... It's a game. Out of that last 10%, 4/5 times it's semi toxic is due to science either fucking off or not exsisting. This is solved shortly by communication or asking the ai. The only last 2% is it's toxic because science has fucking decided to not work together and prosper. This is generally cause by some player deciding not to give a fuck about mining a department they are codependent on, and then proceeding to choose not to communicate. I have not see mining remove ores from orm when science says "someone rushed a tech, we told em off and will get it done asap". If something like that happens usually the toxic relationship is stopped right there and then. It's only when a complete breakdown in communication occurs along with science fucking up tech order. This should rightfully lead to some conflict cause you know it's a game and fuckups should cause problems. I dunno why you guys are literally wanting mining to hide in a bubble more and science to be even more no skill and pointless. Tldr; bad pr

**Comment by redmoogle** [comment]
We can also just yknow remove the meta

**Comment by Mesalikepie** [comment]
Remove the meta and make science more pointless and non meaningful? Remove the meta and make mining less social then whine that miners don't talk enough? Remove the meta so players literally get to communicate less because sometimes people argue? Yeah love this pr so much. Let's remove all radios and headsets so people just don't get to argue and be toxic.

**Comment by redmoogle** [comment]
> 
> 
> Remove the meta and make science more pointless and non meaningful? Remove the meta and make mining less social then whine that miners don't talk enough? Remove the meta so players literally get to communicate less because sometimes people argue? Yeah love this pr so much. Let's remove all radios and headsets so people just don't get to argue and be toxic.

1. thats good in this case. We want them to do whatever for whatever department not meta nodes
2. I'd be glad to hear less miner whining

Seems like you're overly attached to _something_

**Comment by Mesalikepie** [comment]
Red this is a stupid pr. Saying I'm biased doesn't change that fact. 1. A meta will exsist no matter what and you are lying pretending it will disappear with this pr. 2. You and other people are the ones whining miners don't talk enough then encourage it more and then you will complain more. Geez I have to spell it out for you. This pr is B - A - D. Bad

**Comment by ToasterBiome** [comment]
I like the interdepartmental interaction. I literally main mining and science and the only interaction I've seen is just ribbing each other. If you guys are actually getting mad at the interaction you guys are babies lol. In fact I wish we had more department things that relied on each other.

**Comment by Cartlord** [comment]
Honestly, IME I've almost-never seen Sci and mining be toxic to each other, but what I _have_ dealt with is other departments not getting very useful part upgrades (such as Botany not gettig their DNA manip, biogen, or extractor upgraded; or Engi not being able to harvest off turbine for a while because they can't make high-tier freezer parts to cool the exhaust) because Sci dedicates a good chunk of their starting points to Mining and Mining alone.
As for the PR itself, I believe the basic plas cutter and maybe satchel of holding being endor rewards is nice, though I think it'd be right for something as good as the Advanced Plasma Cutter to be Sci-gated.

**Comment by JamieD1** [comment]
Alternative by theos up. #11145
